server/lib/util.py

```diff
@@ -110,6 +114,20 @@ def get_nl_disaster_config():
   return get_subject_page_config(filepath)
 
 
+# Returns dict of place dcid to place type to geojson object
```

Comment:
could you add to the comment if it's right-hand rule (or not), and if it's a complete geojson (we usually have feature collections only)?

Reply:
done